api/controllers/apiConfigController.js

```diff
@@ -0,0 +1,17 @@
+import fs from 'fs'
+
+class ApiConfigController {
```

Comment:
maybe it is better to rename to serviceRouter, what do you think about that?
Bcz it handle request of getting service dependencies, so it's not connected to api config, just to service dependencies

Reply:
but i already have serviceRouter named as apiConfigRouter, maybe i just rename apiConfigRouter and handle all from getDependencies function in lambda function  
"apiConfigRouter.get('/getDependencies', (req, res) => { HERE })"